### ISSUE TITLE
Implement EdX review comments for cookie-consent banner

### ISSUE THREAD

**Issue by lidijarad** [proversity-org/edx-platform#302]
Need to do:
* Checkout en/django.po file from upstream/master (we never need to make changes to these files.)
* Address colour contrast for background/text colour. So far, ```#007db8``` fails WCAG AAA contrast test. (Need to make it darker to pass the test)
* Does the banner need to be keyboard accessible? Was this something that Insites have already built in?
* What can we do about the default window div that uses aria-live?

**Comment by lidijarad** [comment]
Incorrectly created this issue. Closing.
